### DISCUSSION TITLE
Disable index page and keeping edit page

### DISCUSSION THREAD

**Discussion by atenadadkhah** [filamentphp/filament#4143] [Help]
Hi.
I have a resource called `setting` and I want it to be like when I click on this resource, go to the edit page of the first record (because there is just one record in this table) and I don't want the index page of this resource to be shown. How can I do that?

**Comment by danharrin** [comment]
You shouldn't use a resource for this. You should create a Page, and use the form builder. Resources are only built for CRUD interfaces.
